pkg/federate/kubefed/resource_test.go

```diff
@@ -262,7 +263,64 @@ func testDistribute() {
 	})
 }
 
-func newPod(name string, imageName string) *corev1.Pod {
+func testDelete() {
+	var (
```

Comment:
even though these variables are re-assigned everytime, I'd recommend an AfterEach that resets them as safe measure